implementation.md

```diff
@@ -25,3 +25,60 @@ Implementations that support the Transactions Extension or require credentials f
 implement different behavior, for example, allowing credentials when requests are coming from a trusted domain, 
 allowing DELETE, PUT, or PATCH methods, or 
 permitting the `If-Match` request header.
+
+## datetime Parameter Handling
+
+The datetime parameter used by the Item Search and Features conformance classes allows the same values as the 
+[OAF datetime](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_parameter_datetime) parameter. This allows for 
+either a single [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) datetime or an open or closed interval that also 
+uses RFC 3339 datetimes.
+
+RFC 3339 is a profile of ISO 8601, adding these constraints to the allowed values:
+- a complete representation of date and time (fractional seconds optional).
+- requires 4-digit years
+- only allows a period character to be used as the decimal point for fractional seconds
+- requires the zone offset to be `Z` or like `+00:00`, while ISO8601 allows timezones like `+0000`
+
+A simple regex for an RFC 3339 datetime is:
+
+```
+^(\d\d\d\d)\-(\d\d)\-(\d\d)(T|t)(\d\d):(\d\d):(\d\d)([,.]\d+)?(Z|([-+])(\d\d):(\d\d))$
+```
+
+This is not a precise regex, as it matches some strings that violate semantics. There are additional restrictions, for example, 
+the month (01-12), the day (01-31), the hour (0-24), minute (0-60), seconds (0-9), and timezone offsets.  However, the best 
+strategy is to use this regex to ensure the datetime conforms to the the RFC3339 profile and then us an ISO8601 parser to produce
+a valid datetime object from the datetime string.
+
+Thereby, the recommended process for parsing the datetime is:
+
+1. uppercase the string (this avoids needing to match on both (t|T) and (z|Z))
+2. split the string on `/` to determine if it is a single datetime or an interval
+3. For each value of the split, check if it is either an open interval (the empty string or `..`), or if it matches the RFC3339 datetime regex
+5. ISO8601 parse datetime strings using a library such as [pyiso8601](https://github.com/micktwomey/pyiso8601) or 
```

Comment:
Maybe list two libs for different languages instead of two libs for the same language?

Reply:
I've added one that looks good for JavaScript. I mostly didn't want to recommend something that I wasn't sure if it worked or not. The main point was the Python's built-in library doesn't work, but I don't know about any other languages. I'm not even sure the Scala/JVM one worked correctly.